lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_betabuild_prechecks.rb

```diff
@@ -95,7 +95,7 @@ def self.return_value
 
       def self.authors
         # So no one will ever forget your contribution to fastlane :) You are awesome btw!
-        ["loremattei"]
+        ['loremattei']
```

Comment:
Out of scope & Nitpick. Seeing this also made me wonder if we should have a more generic "WordPress|Automattic Mobile Platform Team" user for these? 🤔 

```suggestion
        ['Lorenzo Mattei']
```

Reply:
I thought about it too. (also there's a lot of info missing in most of those actions metadata but I plan to address that separately)

Reply:
Excellent! 👍